src/components/login/LoginForm.js

```diff
@@ -0,0 +1,97 @@
+import  React, {Component} from 'react';
+import  Validator from 'validator';
+import {Form, Button, Message } from 'semantic-ui-react';
+import  PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import InLineError from '../messages/InLineError';
+
+export const validate = data => {
+        const errors = {};
+        if(Validator.isEmpty(data.username)) errors.username = 'username can\'t be blank';
+        if(Validator.isEmpty(data.password)) errors.password = 'Password can\'t be blank';
+        return errors;
+    }
+
+var formInputStyle = {
+    color : 'white',
+    fontSize : '15px',
+    letterSpacing : '1px'
+};
+
+class LoginForm extends Component{
```

Comment:
I think that since this class has it own state it should have a `constructor` to enable you pass data through props to a presentational component and also it assigns the initial `this.state`. you can read about them:
https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0
https://reactjs.org/docs/state-and-lifecycle.html

Reply:
it is also safe to omit the constructor if you do not need a call to `super` or bind `this` in methods. it is possible to define class properties directly.